src/styles/_viewerControls.scss

```diff
@@ -45,11 +45,12 @@
 		position: absolute;
 		left: 0;
 		right: 0;
-		top: 0;
-		height: 5px;
+		top: -1px;
+		height: 10px;
 		width: 100%;
 		transform: translateY(-100%);
 		transition: height .25s;
+		background: rgb(0 0 0 / 0.5);
```

Comment:
I get a compile error with `make dist`, it says:
```
    Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
    SassError: Function rgb is missing argument $green.
            on line 36 of /srv/www/htdocs/owncloud/apps-external/files_mediaviewer/src/styles/default.scss
    >>          background: rgb(0 0 0 / 0.5);
```

I think you should add commas

Reply:
I've fixed it

Reply:
❤️